cggmp21/src/signing.rs

```diff
@@ -365,6 +406,7 @@ async fn signing_t_out_of_n<M, E, L, D, R>(
     S: &[PartyIndex],
     message_to_sign: Option<DataToSign<E>>,
     enforce_reliable_broadcast: bool,
+    #[cfg(feature = "hd-wallets")] additive_shift: Option<Scalar<E>>,
```

Comment:
I think changing the number of parameters depending on enabled features is a terrible idea. Why not make this a mandatory parameter and pass None?

Reply:
This would also simplify the code below where we have to ifdef again to get the value, where we could unconditionally do `unwrap_or`

Reply:
Yeah indeed we can let it be None when feature is disabled

Reply:
But I don't agree that having different number of parameters is "terrible idea". When the function is private and only used in one place, it doesn't matter.

Reply:
Updated